app/models/reports/result.rb

```diff
@@ -198,16 +201,31 @@ def registrations_for_percentage_calculation_with_ltfu(period)
       end
     end
 
-    def registrations_for_percentage_calculation(period, key)
-      if key == :controlled_patients_with_ltfu
-        return registrations_for_percentage_calculation_with_ltfu(period)
+    def registrations_for_percentage_calculation_absolute(period)
```

Comment:
This name is a bit confusing - maybe `cumulative_registrations_for_percentage calculation` ?